README.md

```diff
@@ -30,11 +30,11 @@ For a list of contributors, [click here](https://github.com/OpenEnergyDashboard/
 
 This project is licensed under the MPL version 2.0.
 
-See the full licensing agreement [here](LICENSE.txt)
+See the full licensing agreement [here](License.txt)
 
 ## Contributions ##
 
-We welcome others to contribute to this project by writing code for submission or collaborating with us. Before contributing, please sign our [Contributor License Agreement](https://openenergydashboard.github.io/developer/cla.html). Web pages with [information for developers](https://openenergydashboard.github.io/developer/) is available. If you have any questions or concerns feel free to at engage@OpenEnergyDashboard.org.
+We welcome others to contribute to this project by writing code for submission or collaborating with us. Before contributing, please sign our [Contributor License Agreement](https://openenergydashboard.github.io/developer/cla.html). Web pages with [information for developers](https://openenergydashboard.org/developer/developer/) is available. If you have any questions or concerns feel free to [contact us](https://OpenEnergyDashboard.org/contact/).
```

Comment:
I noticed this still uses the original reference to the cla.

Reply:
Thanks for noticing this. All these edits started because PR #1367 noticed the issue with the CLA link in this file. I just merged development and it should now reflect the proper link.